index.html

```diff
@@ -121,7 +119,7 @@ <h3>Source</h3>
 						</button>
 					</div>
 
-					<code for="htmlEg" id="showHTML" class="html hljs"></code>
+					<code for="grid" id="showHTML" class="html hljs"></code>
```

Comment:
This may impact the indentation shown in the html in view source part. Anyway we need to work on showing the full source code, we can handle this over there.

Reply:
Yeah. I tried to fix something there but couldn't and didn't fully revert changes. Sorry.